modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1837,8 +1839,12 @@ return view.extend({
 					if (bssidval == '1')
 						uci.set('wireless', section_id, 'bssid', bss.bssid);
 				}
-				else if (bss.bssid != null) {
-					uci.set('wireless', section_id, 'bssid', bss.bssid);
+				else {
+					if (ssidval != null)
```

Comment:
I'm a little confused now... bssid should always available right?
Should I put this if in the bssid check or leave as is? 

Or remove all the check as bssid should be always available and ssidval is mandatory?
Or check both at the same time with an and? (think this should be correct)

Reply:
Leave the BSSID logic as-is, do the SSID stuff in parallel.